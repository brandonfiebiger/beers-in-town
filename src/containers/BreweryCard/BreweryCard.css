.BreweryCard {
  border: 2px solid;
  border-radius: 20px; 
  width: 60vw;
  margin-bottom: 2rem;
  height: 10rem;
  color: #f9f9f9;
  text-shadow: 1px 1px #574f4f;
  box-shadow: 0 0 70px #000000;
  background: rgba(25, 25, 25, .7);
  padding: 2rem;
}