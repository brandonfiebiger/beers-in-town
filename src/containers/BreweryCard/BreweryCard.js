import React, { Component } from 'react';
import { Route, withRouter, NavLink } from 'react-router-dom';
import { connect } from 'react-redux';
import PropTypes from 'prop-types';

export class BreweryCard extends Component {
  constructor(props) {
    super(props);
    console.log(props)
  }


  render() {
    const { address_1, rating, city, id, state, name, phone } = this.props
    return(
      <section>
        <h1>{ name }</h1>
        <h4>{ city }, { state }</h4>
        <h5>{ address_1 }</h5>
        <h6>{ phone }</h6>
        <h3>rating {rating} out of 5</h3>
      </section>
    )
  }
}

export default withRouter(connect(null, null)(BreweryCard));
