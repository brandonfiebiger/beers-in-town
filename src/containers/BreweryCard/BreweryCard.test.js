import React from 'react';
import { BreweryCard } from './BreweryCard';
import { shallow, mount } from 'enzyme';


describe('BreweryCard', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<BreweryCard />)

  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})