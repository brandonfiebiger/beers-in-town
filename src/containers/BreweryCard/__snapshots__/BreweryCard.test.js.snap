// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BreweryCard should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <BreweryCard />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1 />,
        <h4>
          , 
</h4>,
        <h5 />,
        <h6 />,
        <h3>
          rating 
          NaN
           out of 5
</h3>,
      ],
      "className": "BreweryCard",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            undefined,
            ", ",
            undefined,
          ],
        },
        "ref": null,
        "rendered": Array [
          undefined,
          ", ",
          undefined,
        ],
        "type": "h4",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "h5",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "h6",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "rating ",
            "NaN",
            " out of 5",
          ],
        },
        "ref": null,
        "rendered": Array [
          "rating ",
          "NaN",
          " out of 5",
        ],
        "type": "h3",
      },
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1 />,
          <h4>
            , 
</h4>,
          <h5 />,
          <h6 />,
          <h3>
            rating 
            NaN
             out of 5
</h3>,
        ],
        "className": "BreweryCard",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              undefined,
              ", ",
              undefined,
            ],
          },
          "ref": null,
          "rendered": Array [
            undefined,
            ", ",
            undefined,
          ],
          "type": "h4",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": "h5",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": "h6",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "rating ",
              "NaN",
              " out of 5",
            ],
          },
          "ref": null,
          "rendered": Array [
            "rating ",
            "NaN",
            " out of 5",
          ],
          "type": "h3",
        },
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
